src/d/actor/d_a_npc_seic.cpp

```diff
@@ -4,601 +4,296 @@
+        J3DMaterial* material = modelData->getMaterialNodePointer(getEyeballMaterialNo());
+        material->setMaterialAnm(matAnm);
+    }
+    return draw(FALSE, TRUE, field_0xde8, NULL, 100.0f, FALSE, FALSE, FALSE);
 }
 
-/* 80AC7A68-80AC7A88 000608 0020+00 1/1 0/0 0/0 .text
- * createHeapCallBack__12daNpc_seiC_cFP10fopAc_ac_c             */
-void daNpc_seiC_c::createHeapCallBack(fopAc_ac_c* param_0) {
-    // NONMATCHING
+/* 80AC7A68-80AC7A88 000608 0020+00 1/1 0/0 0/0 .text            createHeapCallBack__12daNpc_seiC_cFP10fopAc_ac_c */
+int daNpc_seiC_c::createHeapCallBack(fopAc_ac_c* actor) {
+    daNpc_seiC_c* i_this = (daNpc_seiC_c*)actor;
+    return i_this->CreateHeap();
 }
 
 /* 80AC7A88-80AC7AA8 000628 0020+00 1/1 0/0 0/0 .text            getType__12daNpc_seiC_cFv */
-void daNpc_seiC_c::getType() {
-    // NONMATCHING
+u8 daNpc_seiC_c::getType() {
+    switch ((u8)fopAcM_GetParam(this)) {
+        case 0:
+            return TYPE_0;
+    }
+
+    return TYPE_1;
 }
 
 /* 80AC7AA8-80AC7AC4 000648 001C+00 1/1 0/0 0/0 .text            getFlowNodeNo__12daNpc_seiC_cFv */
-void daNpc_seiC_c::getFlowNodeNo() {
-    // NONMATCHING
+u32 daNpc_seiC_c::getFlowNodeNo() {
+    u16 rv = home.angle.x;
+    if (rv == 0xffff) {
+        return -1;
+    }
+    return rv;
 }
 
 /* 80AC7AC4-80AC7ACC 000664 0008+00 1/1 0/0 0/0 .text            isDelete__12daNpc_seiC_cFv */
-bool daNpc_seiC_c::isDelete() {
-    return false;
+int daNpc_seiC_c::isDelete() {
+    return 0;
 }
 
-/* ############################################################################################## */
-/* 80AC92C4-80AC92C8 0000A8 0004+00 0/1 0/0 0/0 .rodata          @4292 */
-#pragma push
-#pragma force_active on
-SECTION_RODATA static f32 const lit_4292 = -1.0f;
-COMPILER_STRIP_GATE(0x80AC92C4, &lit_4292);
-#pragma pop
-
 /* 80AC7ACC-80AC7C1C 00066C 0150+00 1/1 0/0 0/0 .text            reset__12daNpc_seiC_cFv */
 void daNpc_seiC_c::reset() {
-    // NONMATCHING
+    initialize();
+    int size = (u8*)&field_0xe60 - (u8*)&mAction;
+    memset(&mAction, 0, size);
+
+    if (mpMatAnm[0] != NULL) {
+        mpMatAnm[0]->initialize();
+    }
+    
+    setAngle(home.angle.y);
+    mMotionSeqMngr.setNo(0, -1.0f, 0, 0);
 }
 
 /* 80AC7C1C-80AC7D34 0007BC 0118+00 1/0 0/0 0/0 .text            setParam__12daNpc_seiC_cFv */
 void daNpc_seiC_c::setParam() {
     // NONMATCHING
-}
+    selectAction();
+    srchActors();
 
-/* 80AC7D34-80AC7D48 0008D4 0014+00 1/1 0/0 0/0 .text            getDistTable__12dAttention_cFi */
-// void dAttention_c::getDistTable(int param_0) {
-extern "C" void getDistTable__12dAttention_cFi() {
-    // NONMATCHING
+    f32 fVar1 = daNpc_seiC_Param_c::m.field_0x90;
+    dComIfGp_getAttention().getDistTable(0x28).mDistMax = fVar1;
+    dComIfGp_getAttention().getDistTable(0x28).mDistMaxRelease = fVar1;
+
+    dComIfGp_getAttention().getDistTable(0x27).mDistMax = fVar1;
+    dComIfGp_getAttention().getDistTable(0x27).mDistMaxRelease = fVar1;
```

Comment:
```suggestion
    dComIfGp_getAttention().getDistTable(0x28).mDistMax = daNpc_seiC_Param_c::m.field_0x90;
    dComIfGp_getAttention().getDistTable(0x28).mDistMaxRelease = daNpc_seiC_Param_c::m.field_0x90;

    dComIfGp_getAttention().getDistTable(0x27).mDistMax = daNpc_seiC_Param_c::m.field_0x90;
    dComIfGp_getAttention().getDistTable(0x27).mDistMaxRelease = daNpc_seiC_Param_c::m.field_0x90;
```

And mark as `Equivalent # weak func order`. Also note that in the debug rom these are accessed with a member pointer, not the global static variable.

Thanks!